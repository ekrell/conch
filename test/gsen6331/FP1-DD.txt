5.790000000000000000e+02,1.870000000000000000e+02
5.460000000000000000e+02,2.180000000000000000e+02
4.030000000000000000e+02,6.880000000000000000e+02
4.660000000000000000e+02,9.180000000000000000e+02
4.850000000000000000e+02,9.890000000000000000e+02
