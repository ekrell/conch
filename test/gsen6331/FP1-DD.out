Using input region raster: test/gsen6331/full.tif
      input graph file: test/gsen6331/full_evg-b_fp1.pickle
  Start: (42.32343, -70.99428)
    End: (42.336, -70.88737)
Incorporating forces into energy cost:
  u: test/gsen6331/waterMag.tif
  v: test/gsen6331/waterDir.tif
Done solving with a*, 221.7360167503357 seconds
Planning results (with currents):
    Distance: 9.6835 km
    Duration: 32.2784 min
    Cost: 259.0397
